docker/server.Dockerfile.dev

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.7.3-buster
+
+RUN apt update && apt install -y \
+  build-essential  \
+  postgresql \
+  freetds-dev
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY Gemfile Gemfile.lock ./
+RUN gem install bundler && RAILS_ENV=production bundle install --jobs 20 --retry 5
```

Comment:
We can remove this `RAILS_ENV=production` right?

Reply:
yes, its a typo! Good catch